File: Part2/C++/BasciConcepts/BasicConcepts_InvalidASCII.cpp
#include <cstdio>
#include <climits>
#include <cfloat>

int main()
{
    // So the ASCII table shows the tab symbol as TAB, but this doesn't work in programming.
    // I think in some programs this will crash, so it will be in a separate program.
    char charInvalid = 'TAB';
    printf ( "Invalid char: %c\n", charInvalid ) ;

    return 0;
}

File: Part2/C#/BaseConcepts/BasicConcepts_InvalidASCII.cs
using System;

class Program
{
    static void Main()
    {
        // So the ASCII table shows the tab symbol as TAB, but this doesn't work in programming.
        // I think in some programs this will crash, so it will be in a separate program.
        // C# doesn't allow multi-character literals - this will cause compile error
        char charInvalid = 'TAB';
        Console.WriteLine ( "Invalid char: {0}", charInvalid );
    }
}
